src/coreclr/src/utilcode/ex.cpp

```diff
@@ -22,6 +22,7 @@
 #include "corexcep.h"
 
 #define MAX_EXCEPTION_MSG   200
+#define THROWN_BY_US_MARKER ULONG_PTR(-1) // just a non-0 value
```

Comment:
It do not think this will work well on Windows when two CLRs are loaded in the same process. It is not officially supported, but it does happen in practice.

Reply:
I can replace this with module base now.

Reply:
If there are two CLRs can be active in the same process, module-specific marker would make more sense.

Reply:
Yes, this is used in production too,